drivers/Serial.h

```diff
@@ -81,6 +81,23 @@ class Serial : public SerialBase, public Stream {
      */
     Serial(PinName tx, PinName rx, int baud);
 
+    /* Stream gives us a FileHandle with non-functional poll()/readable()/writable. Pass through
+     * the calls from the SerialBase instead for backwards compatibility. This problem is
+     * part of why Stream and Serial should be deprecated.
+     */
+    bool readable()
```

Comment:
should be ``bool readable() const`` , and the rest ``xxxable`` also? Filehandle defines it as such